### PR TITLE
Fix issue type ID overwrite

### PR DESCRIPTION
**Pull request by Jake151**
In the event a Jira project has more than one standard issue type defined but the local config file only has one, the code would overwite the correct jira ID with the wrong one resulting in errors where custom fields don't exist on screens. 

This fix aims to resolve that by comparing the name of the issue type in the config file with the issue type from remote. Additionally it will only take the remote ID when there isn't one defined in the config file.

### CODE REVIEW

**Comment by zbain** [comment]
Tracking this PR, I suspect this is related to a longstanding issue we've been having getting the integration up for some time.

**Review by alpesh-te — APPROVED** [review]
LGMT

**Comment by Jake151** [comment]
Hi @alpesh-te  - I don't have write access to the repo so can't merge this

**Comment by zbain** [comment]
@alpesh-te Can you merge?

**Review by SteveMcGrath — APPROVED** [review]

**Pull request merged**
